pkgs/servers/emby/default.nix

```diff
@@ -30,7 +30,7 @@ stdenv.mkDerivation rec {
     mkdir -p $out/bin
```

Comment:
It would be nice to quote these: `mkdir -p "$out/bin"`. In theory there might be spaces in a custom nix store path and in general quoting variables is just always safer.

Reply:
I agree with you, but I think this is completely unrelated to the package update.

The package might need to be reworked anywaqy in the future to switch from Mono to .NET Core.

Reply:
Yes, but I think we should apply the [boy scout rule](https://www.informit.com/articles/article.aspx?p=1235624&seqNum=6). Otherwise these small improvements will never be made.